core/common/assets/js/components/wordpress/commands-data/elementor-templates.js

```diff
@@ -0,0 +1,33 @@
+import CommandData from 'elementor-api/modules/command-data';
+
+export class ElementorTemplates extends CommandData {
+	static getEndpointFormat() {
+		return 'elementor-templates';
```

Comment:
The CPT is `elementor_library`
```suggestion
		return 'elementor-library';
```